docs/type.md

```diff
@@ -1,98 +1,197 @@
 ---
-title: Type!
+title: Types
 ---
 
-Types are the highlight of Reason! Here, you get a glimpse of why so many are excited about them.
+Types describe what kind of thing values are. Is this value a string, integer,
+or some complex structure? The type of that value will give a direct answer.
```

Comment:
> The type of that value will give a direct answer.
Well, sometimes :D
(Polymorphic types, abstract types).

Reply:
> The type of that value gives an answer.

Is that better? Just less explicit. It does give an answer, just maybe it's not a complete answer like you point out :P